src/main/kotlin/dsm/wemeet/domain/friend/repository/FriendJpaRepository.kt

```
@@ -0,0 +1,8 @@
+package dsm.wemeet.domain.friend.repository
+
+import dsm.wemeet.domain.friend.repository.model.Friend
+import org.springframework.data.jpa.repository.JpaRepository
+import java.util.UUID
```

detekt (1.23.7)

[warning] The class or object CheckFriendService is empty.

(detekt.empty-blocks.EmptyClassBlock)